### ISSUE TITLE
Name of page. Summary of issue

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#2111]
## Description of issue

windows 11
voice access
edge

[Fable recording](https://app.makeitfable.com/share/685ec98a816251063689d19d?video=1 )

Describe what is failing
At 7:08 and 14:20
- Bullet lists are triggering numbers on Voice Access on Edge. 
- Sometimes, CSS and styles can influence AT to behave in a specific way. See if this behavior is standard with Voice Assist on Edge or if we are causing it. 
- In the main nav, see if the doubling is caused by the duplicated nav elements. 

## Potential fix

- What change could fix this issue?
- What nuggets of information could help us fix the issue

## Resources

- Links to documentation or articles that could be helpful
- examples: WCAG docs, ARIA docis, web.dev, Deque, Webaim...

## Acceptance Criteria

- [ ] Should this issue be added to the A11y Statement if it cannot be resolved soon?
